transcom/mymove
# B 21036 enforce dodid and emplid int

## [B-21036](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1007158)

## Summary

Enforced DODID for all affiliations and when Coast Guard also enforces EMPLID. This is a form validation change and endpoint-specific validation change

### How to test

1. Login as a SC
2. Navigate to add a customer (See screenshots below)
3. Create a new customer and test DODID and EMPLID requirements

## Screenshots
![image](https://github.com/user-attachments/assets/82e9c563-e68a-4b3e-9e63-830469ff5699)
![Screenshot 2024-09-06 at 10 59 33 AM](https://github.com/user-attachments/assets/7329b0df-0f6d-407e-8218-b8302ed7af34)
![Screenshot 2024-09-06 at 11 02 11 AM](https://github.com/user-attachments/assets/6b315906-4376-4a45-b67a-75b2a2f94261)
![Screenshot 2024-09-06 at 11 02 59 AM](https://github.com/user-attachments/assets/d0c8d566-522c-4203-ab4c-283da6e2e8f3)

## 508 Screenshot
![Screenshot 2024-09-06 at 11 02 32 AM](https://github.com/user-attachments/assets/716be9ce-00c0-435c-87c0-e7e63a4bcea1)
